api/src/pcapi/core/categories/subcategories_v2.py

```diff
@@ -192,17 +192,17 @@ def get_show_nodes() -> list[ShowGenre]:
 SEARCH_GROUP_ARTS_LOISIRS_CREATIFS = SearchGroup(
     technical_name="ARTS_LOISIRS_CREATIFS",
     label="Arts & loisirs créatifs",
-    position=5,
+    position=6,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** par curiosité, comment on choisit la valeur de position ?
low logaf‏

Reply:
C'est les UX qui décident.
L'ancien search group était en 2, et on veut mettre les deux qui le remplacent au même endroit, donc positions 2 et 3, donc ça décale le reste de 1.
Pour vérifier qu'on fait pas de bêtise, c'est bien de tester en local et de comparer à la prod.
Je pourrai rajouter un screenshot, mais l'app ne prend pas encore en compte ces données-là (WIP).